charters/web5-working-group.md

```diff
@@ -0,0 +1,161 @@
+
+* Implementation of a common set of standards using a well-defined API for the usage of these standards across programming languages.
+
+* Support for the needs of the TBD business unit, including, but not limited to the feature set required by [tbDEX](https://developer.tbd.website/projects/tbdex/).
+
+* Conformance infrastructure such as test vectors, test suites to encourage compatible implementations across programming languages.
+
+* Enable the development of Decentralized Applications (dApps) using technologies such as Decentralized Identifiers (DIDs) and Verifiable Credentials (VCs).
+
+## Deliverables
+
+### 1. Web5 Specification
+
+**Description**: Develop and maintain the [Web5 Specification](https://github.com/TBD54566975/web5-spec/), that clearly specify:
+
+**DRI**: [Gabe Cohen](https://github.com/decentralgabe)
+
+**Scope of Work**:
+
+* Specification of the digital identity standards we will implement (e.g. specific DID methods, verifiable credential data models, credential issuance protocols) along with specific implementation details (e.g. required features, minimal subsets, etc.).
+
+* An API used to interact with the digital identity standards we implement, able to be uptaken across Web5 SDKs.
+
+* Test vectors from which conformance to each feature in the specification can be achieved.
+
+### 2. Web5 SDKs
+
+**Description**: Develop and maintain Web5 SDKs which shall conform to the Web5 Specification.
+
+**DRI**: [Frank Hinek](https://github.com/frankhinek)
+
+**Scope of Work**:
+
+* SDKs in [Typescript](https://github.com/TBD54566975/web5-js), [Kotlin](https://github.com/TBD54566975/web5-kt), [Swift](https://github.com/TBD54566975/web5-swift), [Dart](https://github.com/TBD54566975/web5-dart), [Go](https://github.com/TBD54566975/web5-go), and [Rust](https://github.com/TBD54566975/web5-rs).
+
+* SDKs shall demonstrate conformance to the test vectors defined by the Web5 specification.
+
+* SDKs shall be well-documented according to each language's conventions, as well as being surfaced through the [developer site](https://developer.tbd.website/docs/).
+
+* SDKs shall follow the guidance of the Rust Working Group in adopting a Rust core to limit developer overhead. 
+
+### 3. DID DHT
+
+**Description**: Develop and maintain the DHT Decentralized Identifier method (did:dht).
+
+**DRI**: [Gabe Cohen](https://github.com/decentralgabe)
+
+**Scope of Work**:
+
+* Develop and maintain the [DID DHT Method Specification](https://did-dht.com/) and [DID DHT Method Registry](https://did-dht.com/registry).
+
+* Develop and maintain an open source implementation of a [DID DHT Gateway Server](https://did-dht.com/#gateways).
+
+* Develop and maintain a conformance testing suite for DID DHT clients, including test vectors and implementation reports.
+
+* Ensure conformant implementation of the DID DHT method across **Web5 SDK** implementations.
+
+### 4. Decentralized Web Nodes (DWNs)
```

Comment:
for this section and the one below could use input from @thehenrytsai @csuwildcat @LiranCohen